README.md

```diff
@@ -1,26 +1,28 @@
 # good-errors
+
 Utility stream to transform errors to object literals.
 
 [![Build Status](https://travis-ci.org/benleen/good-errors.svg?branch=master&style=flat)](https://travis-ci.org/benleen/good-errors)
 
 Example usage as hapijs server options:
-```
+
+```javascript
 const options = {
     reporters: {
         consoleReporter: [
-        {
-            module: 'good-squeeze',
-            name: 'Squeeze',
-            args: [{ log: '*', response: '*' }]
-        },
-        {
-            module: 'good-errors'
-        },
-        {
-            module: 'good-console'
-        },
-        'stdout',
-        ],
-    },
-}
-```
\ No newline at end of file
+            {
+                module: '@hapi/good-squeeze',
+                name: 'Squeeze',
+                args: [{ log: '*', response: '*' }]
+            },
+            {
+                module: '@hapi/good-errors'
```

Comment:
This should be `'good-errors'` still if I'm not mistaken.

Reply:
fixed up, pasted @hapi to many times 🤦‍♂